src/index.js

```diff
@@ -140,41 +153,44 @@ module.exports = {
     'no-useless-concat': 'error',
     'no-var': 'error',
     'no-void': 'error',
-    'no-warning-comments': 'off',
-    'no-with': 'off',
-    'object-curly-spacing': 'off',
+    'no-warning-comments': 'error',
+    'no-with': 'error',
+    'object-curly-spacing': ['error', 'always'],
     'object-shorthand': 'error',
-    'one-var': 'off',
+    'one-var': ['error', 'never'],
     'operator-assignment': 'error',
-    'operator-linebreak': 'off',
-    'padded-blocks': 'off',
+    'operator-linebreak': 'error',
+    'padded-blocks': ['error', 'never'],
     'prefer-arrow-callback': 'error',
     'prefer-const': 'error',
     'prefer-spread': 'error',
     'prefer-template': 'error',
-    quotes: 'off',
-    'quote-props': 'off',
+    'quote-props': ['error', 'as-needed'],
+    quotes: ['error', 'single'],
```

Comment:
Do we need to provide `allowTemplateLiterals: true` here to allow us to use templates where needed?